src/wallets/bitcoin.ts

```diff
@@ -134,14 +134,16 @@ export class InternalBitcoinWallet implements BitcoinWallet {
   }
 
   public getNominalBalance(): Big {
-    let satBalance = new Big(0);
-    this.unspentOutputs.forEach((utxo: CsUtxo) => {
-      satBalance = satBalance.add(utxo.value);
-    });
+    const satBalance = Array.from(this.unspentOutputs.values()).reduce(
+      (sum, next) => sum.add(next.value),
+      new Big(0)
```

Comment:
❓ I somehow understand what is going on here (`Array.from` creates an Array out of `unspentOutputs.values()` and reduce iterates over the element grouped by two elements), but why is the `new Big(0)` needed at this point?

Reply:
That is the initial value you want to start the `reduce` with. It is the value of the `sum` variable when `next` is `this.unspentOutputs.values()[0]`.

It also handles the case when you have no values at all to iterate over. Then this one will be the one returned.